docs/_docs/navigation.md

```diff
@@ -0,0 +1,572 @@
+      <li><a href="configuration.html">Configuration</a></li>
+      <li><a href="deployment.html">Deployment</a></li>
+   </ul>
+</div>
+
+In this scenario, we want to pass values from the page's front matter into a `for` loop that contains a variable. When the assigned variable isn't a string but rather a data reference, you must use brackets (instead of curly braces) to refer to the front matter's value.
+
+For more information, see [Expressions and Variables](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#expressions-and-variables) in Liquid's documentation. Brackets are used in places where dot notation can't be used. You can also read more details in this [Stack Overflow answer](http://stackoverflow.com/questions/4968406/javascript-property-access-dot-notation-vs-brackets/4968448#4968448).
+
+## Scenario 6: Applying the active class for the current page
+
+In addition to inserting items from the YAML data file into your list, you also usually want to highlight the current link if the user is viewing that page. You do this by inserting an `active` class for items that match the current page URL.
+
+**CSS**
+```css
+.result li.active a {
+    color: lightgray;
+    cursor: default;
+  }
+```
+**Liquid**
+
+```liquid
+{% raw %}{% for item in site.data.samplelist.docs %}
+    <li class="{% if item.url == page.url %}active{% endif %}">
+    <a href="{{ item.url }}">{{ item.title }}</a></li>
+{% endfor %}{% endraw %}
+```
+
+**Result**
+
+<style>
+.result li.active a {
+    color: lightgray;
+    cursor: default;
+  }
+</style>
+
+<div class="result">
+   <li class=""><a href="introduction.html">Introduction</a></li>
+   <li class=""><a href="configuration.html">Configuration</a></li>
+   <li class="active"><a href="deployment.html">Deployment</a></li>
+</div>
+
+In this case, assume `Deployment` is the current page.
+
+To make sure the `item.url` (stored in the YAML file) matches the `page.url`, it can be helpful to print the `{% raw %}{{ page.url }}{% endraw %}` to the page.
+
+## Scenario 6: Including items conditionally
+
+You might want to include items conditionally in your list. For example, maybe you have multiple site outputs and only want to include the sidebar item for certain outputs. You can add properties in each list item and then use those properties to conditionally include the content.
+
+**YAML**
+```yaml
+docs2_list_title: ACME Documentation
+docs2:
+
+- title: Introduction
+  url: introduction.html
+  version: 1
+
+- title: Configuration
+  url: configuration.html
+  version: 1
+
+- title: Deployment
+  url: deployment.html
+  version: 2
+```
+
+**Liquid**
+
+```liquid
+{% raw %}<ul>
+    {% for item in site.data.samplelist.docs2 %}
+        {% if item.version == 1 %}
+          <li><a href="{{ item.url }}">{{ item.title }}</a></li>
+        {% endif %}
+    {% endfor %}
+</ul>{% endraw %}
+```
+
+**Result**
+
+<div class="result">
+   <ul>
+      <li><a href="introduction.html">Introduction</a></li>
```

Comment:
All these clickable links will generate 404 errors 😭 

Shouldn't we use `#` in href for all examples here?